editor/editor_resource_picker.cpp

```diff
@@ -235,6 +235,11 @@ void EditorResourcePicker::_update_menu_items() {
 				edit_menu->add_icon_item(get_editor_theme_icon(SNAME("Clear")), TTR("Clear"), OBJ_MENU_CLEAR);
 			}
 			edit_menu->add_icon_item(get_editor_theme_icon(SNAME("Duplicate")), TTR("Make Unique"), OBJ_MENU_MAKE_UNIQUE);
+			String modifier = "Ctrl";
+			if (OS::get_singleton()->has_feature("macos") || OS::get_singleton()->has_feature("web_macos") || OS::get_singleton()->has_feature("web_ios")) {
+				modifier = "Cmd";
+			}
```

Comment:
Do we have a standard function to replace these lines? I feel like this is common enough in the editor to warrant some kind of general function.